Assets/Scripts/UI/PickupPup/DogProfile/AffectionMeter.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class AffectionMeter : PPUIElement
+{
+    [SerializeField]
+    Image[] hearts;
+
+    [SerializeField]
+    Sprite emptyHeart;
+    [SerializeField]
+    Sprite halfHeart;
+    [SerializeField]
+    Sprite fullHeart;
+
+    public void setAffection(float newAffection)
+    {
+        int halfHeartIntervals = (int)(newAffection * 2);
```

Comment:
Should use the tuning value for affection increments in some way so this doesn't break if we change how much affection increases by.

Same note for the other hardcoded ones and twos